app/config/config.yml

```diff
@@ -3,6 +3,11 @@ imports:
     - { resource: security.yml }
     - { resource: services.yml }
 
+# Put parameters here that don't need to change on each machine where the app is deployed
+# http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    locale: en
```

Comment:
+1 for removing the `database_driver` parameter altogether and configuring it directly. It should make using `sqlite` a little bit easier I think anyways. I don't think this is a BC break, since the SE is just for *new* projects (and nothing internally currently uses the `database_driver` parameter anyways).

So, let's do it!